influx.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	influxClient "github.com/GeertJohan/influxdb-client-old"
```

Comment:
think you have to use a nightly build to get grafana w/ 0.9. that's ok, could you just doc influx version in README for the next person who might end up trying to use this. thanks